web/src/engine/websites/CookMana.ts

```diff
@@ -0,0 +1,92 @@
+import { Tags } from '../Tags';
+import icon from './CookMana.webp';
+import { Chapter, DecoratableMangaScraper, Page, Manga, type MangaPlugin } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import { FetchWindowScript } from '../platform/FetchProvider';
+
+type APIManga = {
+    id: string,
+    title: string
+}
+
+type APIChapter = {
+    id: string,
+    title: string
+}
+
+type APIPage = {
+    folder: string,
+    folder2: string,
+    folder3: string,
+    id: string,
+    parentId: string,
+    urls: string
+}
+
+@Common.ImageAjax()
+export default class extends DecoratableMangaScraper {
+
+    private readonly cdn1 = 'https://www.11angle.net/';
```

Comment:
There are quite some numbers in these CDNs, are they changing regular? Maybe get them during `Initialize` from the website.

Reply:
Variables are not accessible via FetchWindowScript because of the scope. 
So it would need to fetch a separate js file and parse it.

Website code is made so there is only 2 of them (if not first then second) so i guess its not a big deal.